frontend/packages/rhoas-plugin/src/components/service-list/ServiceListPage.tsx

```diff
@@ -68,32 +91,18 @@ const ServiceListPage: React.FC = () => {
       setSubmitting(false);
     } catch (error) {
       deleteKafkaConnection(name, currentNamespace);
-      setKafkaCreateError(error);
-      setSubmitting(false);
-    }
-  }, [currentNamespace, remoteKafkaInstances, selectedKafka]);
 
-  if (kafkaCreateError) {
-    return (
-      <ServicesErrorState
-        title={t('rhoas-plugin~Failed to create connection')}
-        message={kafkaCreateError + t('rhoas-plugin~Please try again')}
-        actionLabel={t('rhoas-plugin~Go back to Services Catalog')}
-      />
-    );
-  }
+      const connectionErrorObj = {
+        title: t('rhoas-plugin~Failed to create connection'),
+        message: error + t('rhoas-plugin~Please try again'),
```

Comment:
wrong way of mixing error and translation.  I have committed that but we might need to improve it

Reply:
I created an issue similar to this that will address "please try again" because its weird to have that string and then no action to allow the user to try again..
sounds like we should follow up in a separate issue, want to create an issue?

Reply:
Ah.. Try again would be to repeat step after you are back. Most of the try again will be after you address core of the issue that is going to be displayed as dynamic error comming from operator or kube API.

So error is - namespace is gone. Please try again... with different one etc.

Reply:
My take is that retry action is not needed in this context but agreed try again is confusing